test/routes/routes_test.go

```diff
@@ -39,7 +39,8 @@ func TestRoutes(t *testing.T) {
 		c int
 	}{
 		{"linkerd-controller-api", 9},
-		{"linkerd-destination", 4},
+		{"linkerd-destination", 1},
```

Comment:
what does this instance of `linkerd-destination` correspond to?

Reply:
This simply does a count on the number of string occurrences. The deployment is still named `linkerd-destination`. This is why it shows up.